subliminal/providers/subscenter.py

```diff
@@ -191,6 +192,7 @@ def query(self, title, season=None, episode=None):
                         subtitle_key = subtitle_item['key']
                         downloaded = subtitle_item['downloaded']
                         release = subtitle_item['subtitle_version']
+                        h_version = subtitle_item['h_version']
```

Comment:
@Diaoul I've updated the commit and the PEP8 test passes.
The `test_subscenter` looks obsolete and needs a lot of adjustments to pass (it doesn't look like it would pass with the current version as well but I'm speculating here)